.circleci/config.yml

```diff
@@ -546,13 +512,28 @@ jobs:
       - conditionally-launch-build-and-store:
           study_key: fon
 
-  app-build-and-store-job:
+  app-build:
     working_directory: *ng_workspace_path
     executor:
       name: build-executor
+    parameters:
+      study_key:
+        type: string
     steps:
-      - app-build-and-store:
-          study_key: << pipeline.parameters.study_key >>
+      - build:
+          study_key: << parameters.study_key >>
+      - run-linter-check
+      - run-tests
+      - run:
+          name: Short circuit - Don't store build artifacts if branch is a pull request
+          command: |
+            if [[ ${CIRCLE_PULL_REQUEST} ]]; then
+              echo "CIRCLE_PULL_REQUEST: $CIRCLE_PULL_REQUEST"
+              echo "CIRCLE_BRANCH: $CIRCLE_BRANCH"
+              echo "Skip store build"
+              circleci-agent step halt
+            fi
```

Comment:
Don't save build artifact from PR to Google bucket because it should not be used for deploy.